gradlew

```diff
@@ -7,7 +7,7 @@
 # you may not use this file except in compliance with the License.
 # You may obtain a copy of the License at
 #
-#      http://www.apache.org/licenses/LICENSE-2.0
+#      https://www.apache.org/licenses/LICENSE-2.0
```

Comment:
Not related to upgrading Gradle.

Reply:
This actually is related to upgrading Gradle.  It's the Gradle Wrapper 6.3 version of `gradlew`.  The Gradle team upgraded this script, so I upgraded to match what is distributed with Gradle 6.3.  I didn't look at the change, I just ensured it was the latest version.  Even if the comment change isn't really necessary, it's best to be consistent with the latest Gradle.